.example.env

```diff
@@ -27,6 +24,12 @@ VEDA_DB_PUBLICLY_ACCESSIBLE=TRUE
 VEDA_RASTER_ROOT_PATH=
 VEDA_STAC_ROOT_PATH=
 
+VEDA_USERPOOL_ID=
+VEDA_CLIENT_ID=
+VEDA_CLIENT_SECRET=secret
```

Comment:
Where is `VEDA_CLIENT_SECRET` used? 🤔

Reply:
Ah, I think I found it https://github.com/NASA-IMPACT/veda-backend/blob/develop/ingest_api/infrastructure/config.py#L43!